MAUI/DataGrid/Summaries.md

```diff
@@ -11,17 +11,647 @@ documentation: UG
+    </sfgrid:SfDataGrid.GroupColumnDescriptions>
+    <sfgrid:SfDataGrid.GroupSummaryRows>
+        <sfgrid:DataGridSummaryRow ShowSummaryInRow="True" Title="Total Salary: {Salary} for {customerID} members">
+            <sfgrid:DataGridSummaryRow.SummaryColumns>
+                <sfgrid:DataGridSummaryColumn Name="Salary" 
+                                                MappingName="Salary" 
+                                                Format="{}{Sum}"
+                                                SummaryType="DoubleAggregate">
+                </sfgrid:DataGridSummaryColumn>
+                <sfgrid:DataGridSummaryColumn Name="customerID" 
+                                                MappingName="CustomerID" 
+                                                Format="{}{Count}"
+                                                SummaryType="CountAggregate">
+                </sfgrid:DataGridSummaryColumn>
+            </sfgrid:DataGridSummaryRow.SummaryColumns>
+        </sfgrid:DataGridSummaryRow>
+    </sfgrid:SfDataGrid.GroupSummaryRows>
+</sfgrid:SfDataGrid>
+
+{% endhighlight %}
+{% highlight c# %}
+
+    this.dataGrid.GroupSummaryRows.Add(new DataGridSummaryRow()
+    {
+        ShowSummaryInRow = true,
+        Title = "Total Salary: {Salary} for {customerID} members",
+        SummaryColumns = new ObservableCollection<ISummaryColumn>()
+        {
+            new DataGridSummaryColumn()
+            {
+                Name="Salary",
+                MappingName="Salary",
+                SummaryType=SummaryType.DoubleAggregate,
+                Format="{Sum}"
+            },
+            new DataGridSummaryColumn()
+            {
+                Name="customerID",
+                MappingName="customerID",
+                Format="{Count}",
+                SummaryType=SummaryType.CountAggregate
+            }
+        }
+    });
+
+{% endhighlight %}
+{% endtabs %}
+
+
+![DataGrid with group summary](Images\Group-Summary\ShowSummaryInRow.png)
+
+### Displaying summary in the column
+
+The summary information can be displayed in the column by setting the [DataGridSummaryRow.ShowSummaryInRow]() to `false` and by defining summary columns. To calculate summary based on the column, specify the following properties:
+
+1. [DataGridSummaryColumn.MappingName](): Provides MappingName of the column (Property name of data object) that you want to calculate summary.
+2. [DataGridSummaryColumn.SummaryType](): Provides different built-in summary calculation functions for various types.
+3. [DataGridSummaryColumn.Format](): Provides format string for the summary based on support function name in the specified SummaryType.
+
+Refer to [Formatting Summary](#formatting-summary) section to know more about how to format summary and [Aggregate Types](#aggregate-types) section to know about different Summary Types.
+
+In the following code snippet, summary is defined for `Salary` and `CustomerID` columns:
```

Comment:
The summary information can be displayed in a column by setting the [DataGridSummaryRow.ShowSummaryInRow](https://chat.openai.com/) property to false and defining summary columns. To calculate the summary based on a column, you need to specify the following properties:

[DataGridSummaryColumn.MappingName](https://chat.openai.com/): This property provides the MappingName of the column (the property name of the data object) for which you want to calculate the summary.
[DataGridSummaryColumn.SummaryType](https://chat.openai.com/): This property provides different built-in summary calculation functions for various types.
[DataGridSummaryColumn.Format](https://chat.openai.com/): This property allows you to specify a format string for the summary based on the supported function name in the specified SummaryType.
Please refer to the "Formatting Summary" section to learn more about how to format the summary, and the "Aggregate Types" section to understand the different Summary Types.

In the following code snippet, a summary is defined for the Salary and CustomerID columns:

Reply:
Modified.